### PR TITLE
build: release v6.39.0

### PR DESCRIPTION
**Pull request by justynoh**
## New

- fix(AttachmentField): clone attachment to (hopefully) fix NetworkError [`#6042`](https://github.com/opengovsg/FormSG/pull/6042)
- fix: bump downshift from 6.1.12 to 7.3.2, a11y fixes for single select and multiselect [`#5909`](https://github.com/opengovsg/FormSG/pull/5909)
- fix: remove timezone from datepicker [`#6025`](https://github.com/opengovsg/FormSG/pull/6025)
- fix: add staging-alt2 to deployment script [`#6019`](https://github.com/opengovsg/FormSG/pull/6019)
- build: merge release 6.38.0 to develop [`#6017`](https://github.com/opengovsg/FormSG/pull/6017)
- build: release v6.38.0 [`#6004`](https://github.com/opengovsg/FormSG/pull/6004)
- feat: more logging on fetch fallback [`#6008`](https://github.com/opengovsg/FormSG/pull/6008)

## Dependencies

- fix(deps): bump libphonenumber-js from 1.10.24 to 1.10.26 in /shared [`#6033`](https://github.com/opengovsg/FormSG/pull/6033)
- fix(deps): bump type-fest from 3.7.1 to 3.7.2 in /shared [`#6015`](https://github.com/opengovsg/FormSG/pull/6015)

## Dev-Dependencies

- chore(deps-dev): bump @typescript-eslint/parser from 5.57.0 to 5.57.1 in /shared [`#6032`](https://github.com/opengovsg/FormSG/pull/6032)
- chore(deps-dev): bump @typescript-eslint/eslint-plugin from 5.57.0 to 5.57.1 in /shared [`#6031`](https://github.com/opengovsg/FormSG/pull/6031)

### fix: bump downshift from 6.1.12 to 7.3.2, a11y fixes for single select and multiselect [`#5909`](https://github.com/opengovsg/FormSG/pull/5909)

@LinHuiqing 

This PR has been tested by design system. Can be done on chromatic or pushed to staging to test within a form field setting.
- [x] Dropdowns still work normally in form fields.
- [x] Multi-selects still work normally (e.g. in logic module).

### fix: remove timezone from datepicker [`#6025`](https://github.com/opengovsg/FormSG/pull/6025)

@foochifa 

To test that the issue is fixed, [set your device's timezone](https://support.apple.com/en-gb/guide/mac-help/mchlp2996/mac) to a timezone with negative UTC offset.
- [x] Create a form with a datepicker field.
- [x] Open the form as a respondent and use the datepicker. The selected dates should be correctly reflected in the fields. Submit the form with the selected date on the datepicker.
- [x] As an admin, check the responses. The latest response should have the right date indicated on the datepicker by the respondent.
- [x] As an admin, edit the datepicker and try to set validation with a custom date range. The dates selected for the date range should be reflected correctly as well.

To test the fix did not compromise on any existing datepicker functionality, change the timezone to one with a positive UTC offset and repeat the tests.
- [x] Open the form as a respondent and use the datepicker. The selected dates should be correctly reflected in the fields. Submit the form with the selected date on the datepicker.
- [x] As an admin, check the responses. The latest response should have the right date indicated on the datepicker by the respondent.
- [x] As an admin, edit the datepicker and try to set validation with a custom date range. The dates selected for the date range should be reflected correctly as well.
- [x] For email mode form, disallow past dates / future dates validation should still work based on server time even if client timezone changes

### CODE REVIEW

**Review by timotheeg — APPROVED** [review]

**Review by LinHuiqing — APPROVED** [review]
Merge when https://github.com/opengovsg/FormSG/pull/6025 is tested and ready for release!

**Comment by foochifa** [comment]
Should add one more test for #6025
- [ ] after editing date range, open and make form submission. The dates not reflected in the range should be crossed out accordingly

thus a bug (seems to be replicable regardless of time zone)

image
image
26 is available for selection

image

But can't be submitted

**Comment by LinHuiqing** [comment]
@foochifa I can't seem to replicate the bug you've observed? :O

**Comment by foochifa** [comment]
Yikes, let me share my form https://staging.form.gov.sg/642e2154235f7d00121fa7ac

Or is it my mac thats old

**Comment by justynoh** [comment]
this is what i see 
image

**Comment by LinHuiqing** [comment]
tl;dr: yes there is a bug with the date picker now (when custom date ranges are selected in SGT(?)) and we will be excluding it from this release.

**Comment by justynoh** [comment]
closing to recut
